src/mbgl/annotation/annotation_manager.cpp

```diff
@@ -1,7 +1,7 @@
 #include <mbgl/annotation/annotation_manager.hpp>
 #include <mbgl/annotation/annotation_tile.hpp>
-#include <mbgl/style/style.hpp>
 #include <mbgl/layer/symbol_layer.hpp>
+#include <mbgl/style/style.hpp>
```

Comment:
Yeah, we don't really have a strict system, except that we try to group by path prefix.